src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -276,6 +276,34 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
     // Show spinner since the network calls below can take a few seconds.
     this.$root.$emit('showSpinner', true)
 
+    // Special case to handle Extra-pro A companies
+    if (businessLookup.legalType === 'A') {
+      const tingBusiness = {
+        type: AmlTypes.FOREIGN,
+        role: AmlRoles.AMALGAMATING,
+        foreignJurisdiction: {
+          region: 'British Columbia',
+          country: 'CA'
```

Comment:
Use enums, eg, `JurisdictionLocation.CA`.

Ask Leo if there's a enum (or list data for British Columbia).

Reply:
Region is still hard-coded. Waiting for Leo's answer. Will try to fix the region in next commit.